build-parent/pom.xml

```diff
@@ -22,19 +22,10 @@
         <!-- These properties are not in the bom as we don't want to enforce the version -->
         <spring.version>5.1.4.RELEASE</spring.version>
 
-        <!-- These properties are needed in order for them to be resolvable by the generated projects -->
-        <kotlin.version>1.3.21</kotlin.version>
-
         <!-- These properties are needed in order for them to be resolvable by the documentation -->
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
         <graal-sdk.version-for-documentation>19.0.2</graal-sdk.version-for-documentation>
-        <rest-assured.version>3.3.0</rest-assured.version>
-        <axle-client.version>0.0.7</axle-client.version>
-        <vertx.version>3.7.1</vertx.version>
-
-        <!-- Enable APT by default for Eclipse -->
-        <m2e.apt.activation>jdt_apt</m2e.apt.activation>
```

Comment:
Mmmmh, where did you move these properties? I don't see them anymore (and they are needed)?

Reply:
Ugh, my bad. I had another pom which I decided not to include here. Thanks for catching it @gsmet !